### PR TITLE
1.27.2 announcement

### PR DESCRIPTION
**Pull request by Mark-Simulacrum**
r? @steveklabnik 
cc @rust-lang/core @rust-lang/release

### CODE REVIEW

**Comment by pietroalbini** [comment]
We should add a section about the recent increase of points releases (mostly the improvements to the release process, but also the maintenance problems of the current borrow checker).

There were a few questions on internals and reddit when we first talked about 1.27.2 on the safety/stability of the language with all those patches, and I expect even more when the actual release happens.

File: _posts/2018-07-20-Rust-1.27.2.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+1.27.2][notes] on GitHub.
+
+[install]: https://www.rust-lang.org/install.html
+[notes]: https://github.com/rust-lang/rust/blob/stable/RELEASES.md#version-1272-2018-07-20
+
+## What's in 1.27.2 stable
+
+This patch release fixes a bug in the borrow checker verification of `match` expressions. This bug
+was introduced in 1.26.0 with the stabilization of [match ergonomics]. Specifically, it permitted
+code which took two mutable borrows of the `bar` path at the same time.
```
Comment (dtolnay) [inline _posts/2018-07-20-Rust-1.27.2.md]: Based on this sentence I would expect to see something called `bar` and some mutable borrows in the code below.

Comment (Mark-Simulacrum) [reply _posts/2018-07-20-Rust-1.27.2.md]: Heh, thanks. Maybe copy/pasting the file from 1.26.2 wasn't the best idea...

**Comment by Mark-Simulacrum** [comment]
Pushed fixes.

Would appreciate review on the discussion of reasoning for increased patch releases.

File: _posts/2018-07-20-Rust-1.27.2.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+Users have expressed concern with the frequency of patch releases to fix bugs in the match
+ergonomics verification by the current borrow checker on a variety of Rust's forums. There are two
+primary reasons for the increased rate of patch releases: significantly higher bandwidth and the
+age of the currently used borrow checker. First off, with the formation of a Release team, Rust's
+ability to generate patch releases has greatly increased. This means that the investment from the
+compiler and core teams required to make a patch release is greatly reduced, which also makes such a
+patch release more likely to happen. The current borrow checker has been around for years now, and
+is beginning to show its age. The work on a better, more precise borrow checker is underway,
+and it has detected all of these bugs. This work is planned to be stabilized in the 2018 edition, so
```
Comment (steveklabnik) [inline _posts/2018-07-20-Rust-1.27.2.md]: We have not mentioned the 2018 edition at all in this venue; we shouldn't reference it until we announce it, IMO. So just 

> This work is planned to be stabilized in the next few releases

IMO

File: _posts/2018-07-20-Rust-1.27.2.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+```
+
+1.27.2 will reject the above code.
+
+## Concern over numerous patches to the match ergonomics feature
+
+Users have expressed concern with the frequency of patch releases to fix bugs in the match
+ergonomics verification by the current borrow checker on a variety of Rust's forums. There are two
+primary reasons for the increased rate of patch releases: significantly higher bandwidth and the
+age of the currently used borrow checker. First off, with the formation of a Release team, Rust's
```
Comment (pietroalbini) [inline _posts/2018-07-20-Rust-1.27.2.md]: Maybe we could use a list for "release team"/"borrow checker"? This paragraph is huge.

**Comment by Mark-Simulacrum** [comment]
Split up paragraph and removed note about edition, though we have talked about in the roadmap post I agree it might be best not to mention it.

**Pull request merged**